csgc-jekyll-src/_config.yml

```
@@ -32,6 +32,8 @@ main_pages:
     title: "Officers"
   - url: "/Housing/"
     title: "Housing"
+  - url: "/labrepresentative/"
+    title: "Lab Representative"
```

```suggestion
    title: "Lab Representatives"
```
You might also change the url for the sake of consistency.